### ISSUE TITLE
Evolving multiple pokemon using IDs

### ISSUE THREAD

**Issue by WitherredAway** [poketwo/poketwo#418]
https://github.com/poketwo/poketwo/blob/ec9888296e9e7f315aca2b2b84cdceda691a4842/cogs/pokemon.py#L1157
On this line, wouldn't it be better if it had a `continue` statement after ctx.send, instead of `return`, so that it doesn't stop all the following evolutions aswell?

**Comment by WitherredAway** [comment]
I can create a PR for this upon approval

**Comment by tazzy989** [comment]
Any chance something similar could be done for mass market removal?

**Comment by WitherredAway** [comment]
> Any chance something similar could be done for mass market removal?

Yeah, although I'm not sure if the developer is willing to add more features to the current version of poketwo seeing that the rewrite is incoming
